tests/conftest.py

```diff
@@ -553,3 +553,8 @@ def evaluation_dataset(meta_partitions_evaluation_files_only, store_session):
             dataset_uuid="evaluation_uuid",
             store=store_session,
         )
+
+
+@pytest.fixture
+def dataset_factory_function_scope(store_factory):
```

Comment:
Can you please define this in the test_repartition.py? For now this doesn't make sense globally. In particular, there doesn't exist a dataset with UUID "test" but this is created in the test cases explicilty. A fixture should make sense even outside of the context of a particular test case but this is not the case here

Reply:
done. please verify.